### DISCUSSION TITLE
WalletConnect v2 not working in production

### DISCUSSION THREAD

**Discussion by devsalman247** [wevm/wagmi#2251] [Q&A]
We're using wagmi connectors to provide users to be able with connecting multiple wallets of their choice like MetaMask, Coinbase, Mobile Wallet etc.. Here is the ss of error shown in production on console:
![walletconnect provider error](https://user-images.githubusercontent.com/112937676/235629429-9bb6b77c-629f-4284-96d4-2c8c357e0e45.png)

`import { createClient, configureChains, mainnet } from "wagmi";
import { publicProvider } from "wagmi/providers/public";
import { MetaMaskConnector } from "wagmi/connectors/metaMask";
import { CoinbaseWalletConnector } from "wagmi/connectors/coinbaseWallet";
import { WalletConnectConnector } from "wagmi/connectors/walletConnect";
import { bsc } from "wagmi/chains";

export const walletConnectProjectId = ".....";

const { chains, provider, webSocketProvider } = configureChains([mainnet, bsc], [publicProvider()]);

export const client = createClient({
	autoConnect: true,
	connectors: [
		new MetaMaskConnector({ chains }),
		new CoinbaseWalletConnector({
			chains,
			options: {
				appName: "NFT Promo",
			},
		}),
		new WalletConnectConnector({
			chains,
			options: {
				projectId: walletConnectProjectId,
			},
		}),
	],
	provider,
	webSocketProvider,
})`
This code is working fine and opening modal in local environment but not on production. I'm using **wagmi@0.12.12**

**Comment by jewelsonn** [comment]
That's walletConnect v2 web3modal requirement

**Comment by wmahmood1984** [comment]
Dear could you get the answer of your question?

**Comment by telchardev** [comment]
Same issue in production mode. Error in the console and the modal window is not shown. Everything is ok in dev mode.

**Comment by kveerapen** [comment]
Did anyone find a solution to this?

**Reply by wmahmood1984** [reply]
plz see whether the url used is correct or not
